taxi_demo/gpu_environment.yml

```diff
@@ -0,0 +1,59 @@
+name: saturn
+channels:
+- https://conda.saturncloud.io/pkgs
+- nvidia
+- rapidsai
+- pytorch
+- numba
+- defaults
+- conda-forge
+- saturncloud
+dependencies:
+- blas=*=mkl
+- python=3.7
+- numpy
+- pandas
+- ipywidgets
+- matplotlib
+- dask=2.20.0
+- distributed=2.20.0
+- dask-saturn=0.0.4
+- xgboost
+- dask-xgboost
+- dask-ml
+- rapids=0.14.1=cuda10.1_py37_0
+- numba=0.49.1
+- llvmlite=0.32.1
+- snowflake-connector-python
+- snowflake-sqlalchemy
+- azure-storage-blob=2.1.0
+- scikit-learn
+- scipy
+- bokeh
+- h5py
+- ipykernel
+- graphviz
+- s3fs
+- dash
+- dash-bootstrap-components
+- dash-core-components
+- dash-daq
+- dash-html-components
+- dash-renderer
+- dash-table
```

Comment:
I would expect these envs to be the minimum to support the demo. Do you have a different goal?

Reply:
Yeah I can pare them down, I was just being lazy and copied over what I had in my image